### PR TITLE
Addisie

### PR DESCRIPTION
**Pull request by Addisie**
https://github.com/BetaMasaheft/Documentation/issues/2382 
The name Mǝhǝrkā Dǝngǝl was mentiond in EMML2101 however, I don't know that the same persion was the writer of MDA003.
Sould this persion mentioned in work recored? can he be identified with PRS6978Meherka?

### CODE REVIEW

File: new/LIT6843Commentary.xml [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+            <change who="AY" when="2023-06-27">Discribe the conetnts  </change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+            <div type="bibliography"><bibl>
+                <ptr target="bm:EMML6"></ptr>
+                <citedRange unit="page">196</citedRange>
+            </bibl></div>
+            <div type="edition">
```
Comment (DenisNosnitsin1970) [inline new/LIT6843Commentary.xml:59]: Two points: I would take as the reference manuscript EMML 2101 which is much older and closer to the original (?) than MDA003, and is already described by Getatchew Haile. He calles the collection "Geez commentary on the Pentateuch", that is, all books up up to "Numbers". All texts behind this one in MDA003 looks like additions. Also, looking at  "Maṣḥafa mənhar" I do not see that it is a commentary.  "Maṣḥafa miṭata bərhānāta samāy" is also not a commentary, probably they deserve separate treatment (IDs?). For the rest (Commentary to Maccabees and Daniel) please check if we already have them.  I remember I have recently created records for commentaries on prophetic books. You need to look up in A. Butts' recent articles on EMML 2101, and also in Cowley 1988 on biblical interpretation.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT6843Commentary.xml [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+                    <label>Commentary on Numbers</label>
+                </div>
+                <div type="textpart" xml:lang="gez" xml:id="CommentaryDeuteronomy">
+                    <label>Commentary on Deuteronomy</label>
+                </div>
+                <div type="textpart" xml:lang="gez" xml:id="CommentaryMashafaM̆enhar">
+                <label>Commentary on Maṣḥafa M̆ənhar</label>
+            </div>
+                <div type="textpart" xml:lang="gez" xml:id="CommentaryMashafaM̆itataBrehanataSamey">
+                    <label>Commentary on Maṣḥafa M̆iṭata Bərhānāta Samāy</label>
```
[the remark this answers is not in the document]
Comment (CarstenHoffmannMarburg) [reply new/LIT6843Commentary.xml:83]: Same on Maṣḥafa M̆ənhar = Maṣḥafa mənhar

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: new/LIT6849Menhar.xml [hunk 3]
```diff
@@ -0,0 +1,55 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6849Menhar" xml:lang="en" type="work">
```
Comment (CarstenHoffmannMarburg) [inline new/LIT6849Menhar.xml:4]: A translation to English and a title in fidal would be nice.

File: new/LIT6849Menhar.xml [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation/>
+            <abstract>
+                <p/>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT6849Menhar.xml:33]: Is it possible to write a very short note e.g. one sentence on the content as an abstract?

File: new/LIT6849Menhar.xml [hunk 5]
```diff
@@ -0,0 +1,55 @@
...
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="AY" when="2023-07-07">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
```
Comment (CarstenHoffmannMarburg) [inline new/LIT6849Menhar.xml:51]: Incipit and explicit could be added as well here.

**Review by CarstenHoffmannMarburg — CHANGES_REQUESTED** [review]

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: new/LIT6849Menhar.xml [hunk 6]
```diff
@@ -0,0 +1,55 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6849Menhar" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">Maṣḥafa Mənhar</title>
```
Comment (DenisNosnitsin1970) [inline new/LIT6849Menhar.xml:8]: Please add fidel and translation (Mənhar should be small, and is it ምንሀር or ምንሃር?)

File: new/LIT6843Commentary.xml [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+                    <ptr target="bm:Cowley1988Exeget"></ptr>
+                <citedRange unit="page">114; 118; 432; 438</citedRange>
+                </bibl>
+                <bibl>
+                    <ptr target="bm:Butts2013Reception"></ptr>
+                    <citedRange unit="page">149-151</citedRange>
+                </bibl>
+                </div>
+            <div type="edition">
+                <note>The following edition is based on <ref type="mss" corresp="MDA003"/>.</note>
```
Comment (DenisNosnitsin1970) [inline new/LIT6843Commentary.xml:68]: As I indicated above, it is better to follow EMML 2101; MDA003 can be included through relations.

**Review by DenisNosnitsin1970 — CHANGES_REQUESTED** [review]

File: new/LIT6849Menhar.xml (hunk 6 above)
Comment (Addisie) [reply new/LIT6849Menhar.xml:8]: The file "Maṣḥafa Mənhar" was sent to you via technical errors. I will add all the necessary information.

**Review by CarstenHoffmannMarburg — APPROVED** [review]

File: new/LIT6849Menhar.xml [hunk 8]
```diff
@@ -0,0 +1,62 @@
+<?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+schematypens="http://relaxng.org/ns/structure/1.0"?><?xml-model href="https://raw.githubusercontent.com/BetaMasaheft/schema/master/tei-betamesaheft.rng" 
+type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
+<TEI xmlns="http://www.tei-c.org/ns/1.0" xml:id="LIT6849Menhar" xml:lang="en" type="work">
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="gez" xml:id="t1">መጽሐፈ ምንሀር </title>
+                <title xml:lang="gez" xml:id="t2">Maṣḥafa mənhar</title>
```
[the remark this answers is not in the document]
Comment (eu-genia) [reply new/LIT6849Menhar.xml:9]: PS sorry I see it was @thea-m who did the merge a bit too fast this time )

File: new/LIT6843Commentary.xml (hunk 7 above)
Comment (eu-genia) [reply new/LIT6843Commentary.xml:68]: this comment has not been respected yet.

for the work, please refer to EMML2101 ms. 

MDA003 is secondary, you do not need to mention it in the work record (or add in the witness list)

**Pull request merged**